targets/ipq40xx-generic

```diff
@@ -96,6 +96,14 @@ device('gl.inet-gl-b1300', 'glinet_gl-b1300', {
 device('linksys-ea6350v3', 'linksys_ea6350v3')
 
 
+-- Meraki
+
+device('meraki-mr33', 'meraki_mr33', {
+	factory = false,
+	aliases = {'meraki-mr33-access-point'},
```

Comment:
Why this alias?

Reply:
Because it is returned by `lua -e 'print(require("platform_info").get_image_name())'`.
Isn't that whats needed in this case?

Reply:
a manifest-alias should suffice for the autoupdater.

Reply:
Right, forgot about `manifest_aliases` parameter - thanks!